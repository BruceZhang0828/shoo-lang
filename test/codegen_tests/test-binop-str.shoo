string x = "Hello "+"World"+"!";
println(x);