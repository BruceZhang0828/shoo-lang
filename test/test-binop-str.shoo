string x = "Hello "+"World"+"!";
println(x);

string y = string_concat("Hello ","World");
y = string_concat(y, "!");
println(y);

if (x == y) {
     println("yay");
} else {
    println("not yay");
}
