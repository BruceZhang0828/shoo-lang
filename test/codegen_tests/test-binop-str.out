Hello World!
Hello World!
yay
